qiita_db/user.py

```diff
@@ -266,9 +263,9 @@ def verify_code(cls, email, code, code_type):
         elif code_type == 'reset':
             column = 'pass_reset_code'
         else:
-            raise IncompetentQiitaDeveloperError("code_type must be 'create'"
-                                                 " or 'reset' Uknown type "
-                                                 "%s" % code_type)
+            raise IncompetentQiitaDeveloperError(
+                "code_type must be 'create' or 'reset' Uknown type %s" %
```

Comment:
There could potentially be other codes, for example API keys, so it was left open ended.